articles/quickstart/spa/angular2/_includes/_centralized_login.md

```diff
@@ -37,15 +46,21 @@ export class AuthService {
 }
 ```
 
-::: note
-**Checkpoint:** Try calling the `login` method from somewhere in your application. This could be from a button click or in some lifecycle event, just something that will trigger the method so you can see the login page.
+::: panel Checkpoint
+Try calling the `login` method from somewhere in your application to see the login page.
 :::
 
 ![hosted login](/media/articles/web/hosted-login.png)
 
-### Finish Out the Service
+### Finish the Service
+
+Add more methods to the `AuthService` service to handle authentication in the app.
 
-Add some additional methods to the `AuthService` to fully handle authentication in the app.
+The example below shows the following methods:
+* `handleAuthentication` - looks for the result of authentication in the URL hash and processes it with the `parseHash` method from auth0.js
+* `setSession` - sets the user's access token, ID token, and the access token's expiry time 
```

Comment:
Use this format for consistency:
      * `setSession`: sets the user's access token